qiskit/aqua/operators/primitive_ops/circuit_op.py

```diff
@@ -212,6 +212,10 @@ def to_circuit_op(self) -> OperatorBase:
     def to_instruction(self) -> Instruction:
         return self.primitive.to_instruction()  # type: ignore
 
+    # def to_gate(self) -> Gate:
```

Comment:
Why is this commented out? 🙂

Reply:
I just added skeleton comments to the operators that I think will need this method for later on 😅